src/App.jsx

```diff
@@ -45,6 +44,7 @@ const LogoQuestionValidator = React.lazy(() =>
   import("./pages/logosValidator/LogoQuestionValidator")
 );
 const DashBoard = React.lazy(() => import("./pages/logosValidator/DashBoard"));
+const Loader = React.lazy(() => import("./pages/loader"));
```

Comment:
I would import this statically, as it should be only a few Bytes and using a dynamic import for a fallback seems counterintuitive.

Reply:
@VaiTon there is index.jsx page in pages folder which is importing all the pages ! I think we don't need that file anymore as we are importing everything directly in the app.jsx file.

Reply:
i had imported everywhere statically, this ones left, i will change that

Reply:
I mean this file : https://github.com/openfoodfacts/hunger-games/blob/master/src/pages/index.jsx

Reply:
Okay ! Cool ! Shall I open a issue ?

Reply:
You could even directly open a PR

Reply:
Umm cool !

Reply:
@VaiTon the pr is ready :)